docfx_yaml/extension.py

```diff
@@ -322,6 +322,10 @@ def build_finished(app, exception):
                 # Skip objects without a module
                 continue
 
+            if filename.lower().find('.tests') != -1:
+                # Skip *.tests*.yml files
+                continue
+
```

Comment:
@ericholscher Agree with you. A small question: how Sphinx filter out these tests? From the CNTK's websites, I don't find these tests pages, and I don't find some config to explicit exclude them.